yt_dlp/extractor/screenrec.py

```diff
@@ -0,0 +1,34 @@
+from .common import InfoExtractor
+from ..utils import url_or_none
+
+
+class ScreenRecIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?screenrec\.com/share/(?P<id>[A-Za-z0-9]{10})'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?screenrec\.com/share/(?P<id>\w{10})'
```

Reply:
Imo. matching a bit more broadly here is not a problem, though I don't feel particularly strong about this. If you prefer `a-zA-Z0-9` you can keep it.